src/node_mime.cc

```diff
@@ -0,0 +1,369 @@
+#include <string>
+#include "node.h"
+#include "env.h"
+#include "node_mime.h"
+#include "node_internals.h"
+namespace node {
+
+namespace mime {
+
+using v8::Array;
+using v8::Context;
+using v8::Function;
+using v8::FunctionCallbackInfo;
+using v8::FunctionTemplate;
+using v8::HandleScope;
+using v8::Isolate;
+using v8::Local;
+using v8::Object;
+using v8::String;
+using v8::Value;
+
+
+// utilities for
+namespace {
+
+bool is_ascii_upper_alpha(char c) {
+  return c >= '\x41' && c <= '\x5A';
+}
+bool is_ascii_lower_alpha(char c) {
+  return c >= '\x61' && c <= '\x7A';
+}
+bool is_ascii_digit(char c) {
+  return c >= '\x30' && c <= '\x39';
+}
+bool is_ascii_alpha(char c) {
+  return is_ascii_upper_alpha(c) || is_ascii_lower_alpha(c);
+}
+bool is_ascii_alphanumeric(char c) {
+  return is_ascii_digit(c) || is_ascii_alpha(c);
+}
+char to_ascii_lower(char c) {
+  if (!is_ascii_upper_alpha(c)) return c;
+  return c + 0x20;
+}
```

Comment:
This is basically the same as `ToLower()` from `util.h`, right?

Reply:
seems so,  deleted